client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -193,7 +195,7 @@ class ActivityLogDay extends Component {
 			<ActivityLogItem
 				className={ hasBreak ? 'is-before-dialog' : '' }
 				applySiteOffset={ applySiteOffset }
-				disableRestore={ disableRestore }
+				disableRestore={ disableRestore || 'active' !== rewindState.state }
```

Comment:
why isn't this taken into account with `disableRestore` already?

Reply:
Fixed https://github.com/Automattic/wp-calypso/pull/20528/commits/91eea903e98ce1bff9ed236cb17fc6dc94584f6a